daprdocs/content/en/dotnet-sdk-docs/dotnet-actors/_index.md

```diff
@@ -2,9 +2,447 @@
+
+You can use `ActorProxy.Create<IMyActor>(..)` to create a strongly-typed client and invoke methods on the actor.
+
+Paste the following code into `Program.cs` in the `MyActorClient` project:
+
+```csharp
+using System;
+using System.Threading.Tasks;
+using Dapr.Actors;
+using Dapr.Actors.Client;
+using MyActor.Interfaces;
+
+namespace MyActorClient
+{
+    class Program
+    {
+        static async Task MainAsync(string[] args)
+        {
+            Console.WriteLine("Startup up...");
+
+            // Registered Actor Type in Actor Service
+            var actorType = "MyActor";
+
+            // An ActorId uniquely identifies an actor instance
+            // If the actor matching this id does not exist, it will be created
+            var actorId = new ActorId("1");
+
+            // Create the local proxy by using the same interface that the service implements.
+            //
+            // You need to provide the type and id so the actor can be located. 
+            var proxy = ActorProxy.Create<IMyActor>(actorId, actorType);
+
+            // Now you can use the actor interface to call the actor's methods.
+            Console.WriteLine($"Calling SetDataAsync on {actorType}:{actorId}...");
+            var response = await proxy.SetDataAsync(new MyData()
+            {
+                PropertyA = "ValueA",
+                PropertyB = "ValueB",
+            });
+            Console.WriteLine($"Got response: {response}");
+
+            Console.WriteLine($"Calling GetDataAsync on {actorType}:{actorId}...");
+            var savedData = await proxy.GetDataAsync();
+            Console.WriteLine($"Got response: {response}");
+        }
+    }
+}
+```
+
+## Running the code
+
+The projects that you've created can now to test the sample.
+
+1. Run MyActorService
+
+    Since `MyActorService` is hosting actors, it needs to be run with the Dapr CLI.
+
+    ```bash
+    cd MyActorService
+    dapr run --app-id myapp --app-port 5000 --dapr-http-port 3500 -- dotnet run
+    ```
+
+    You will see commandline output from both `daprd` and `MyActorService` in this terminal. You should see something like the following, which indicates that the application started successfully.
+
+    ```txt
+    ...
+    ℹ️  Updating metadata for app command: dotnet run
+    ✅  You're up and running! Both Dapr and your app logs will appear here.
+
+    == APP == info: Microsoft.Hosting.Lifetime[0]
+
+    == APP ==       Now listening on: https://localhost:5001
+
+    == APP == info: Microsoft.Hosting.Lifetime[0]
+
+    == APP ==       Now listening on: http://localhost:5000
+
+    == APP == info: Microsoft.Hosting.Lifetime[0]
+
+    == APP ==       Application started. Press Ctrl+C to shut down.
+
+    == APP == info: Microsoft.Hosting.Lifetime[0]
+
+    == APP ==       Hosting environment: Development
+
+    == APP == info: Microsoft.Hosting.Lifetime[0]
+
+    == APP ==       Content root path: /Users/ryan/actortest/MyActorService
```

Comment:
```suggestion
    == APP == info: Microsoft.Hosting.Lifetime[0]
    == APP ==       Now listening on: https://localhost:5001
    == APP == info: Microsoft.Hosting.Lifetime[0]
    == APP ==       Now listening on: http://localhost:5000
    == APP == info: Microsoft.Hosting.Lifetime[0]
    == APP ==       Application started. Press Ctrl+C to shut down.
    == APP == info: Microsoft.Hosting.Lifetime[0]
    == APP ==       Hosting environment: Development
    == APP == info: Microsoft.Hosting.Lifetime[0]
    == APP ==       Content root path: /Users/ryan/actortest/MyActorService
```

Reply:
This is what Dapr outputs. I logged an issue for it.